inst/schema/ProgressPhase.schema.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
we should think about where to put the schema for this if we're going to use it elsewhere more commonly - this one corresponds to what we discussed on slack?

Reply:
Yeah this is the general one. Expecting a standard Progress message to be an array of these.